reascripts/ReaSpeech/.luacheckrc

```diff
@@ -8,8 +8,11 @@ globals = {
   "Polo",
   "ReaUtil",
   "table",
+  "Plugin",
+  "PluginActions",
+  "PluginControls",
 }
 
 allow_defined = true
 color = false
-ignore = {"212/self", "212/_.*", "432/self", "631"}
+ignore = {"212/self", "212/_.*", "432/self", "512", "631"}
```

Comment:
We could instead use `-- luacheck: ignore` at the location that's getting flagged. Unreachable code detection seems like it could catch some bugs.